Resources/Prototypes/Entities/Stations/nanotrasen.yml

```diff
@@ -0,0 +1,33 @@
+﻿- type: entity
+  id: BaseStationNanotrasen
+  components:
+    - type: Faction
+      factions:
+        - NanoTrasen
+
+- type: entity
+  id: StandardNanotrasenStation
+  parent:
+    - BaseStation
+    - BaseStationCargo
+    - BaseStationJobsSpawning
+    - BaseStationRecords
+    - BaseStationArrivals
+    - BaseStationEvacuation
+    - BaseStationAlertLevels
+    - BaseStationExpeditions
+    - BaseStationAllEventsEligible
+    - BaseStationNanotrasen
```

Comment:
Nice! I didn't know the parent system allowed multiple inheritance. Good approach.